cmd/azmount/filemanager/azure.go

```diff
@@ -30,19 +30,19 @@ func tokenRefresher(credential azblob.TokenCredential) (t time.Duration) {
 	// JWT tokens comprise three fields. the second field is the payload (or claims).
 	// we care about the `aud` attribute of the payload
 	curentTokenFields := strings.Split(currentToken, ".")
```

Comment:
typo for `curentTokenFields` -> `currentTokenFields`

Reply:
fixed, thanks!